mycroft/skills/main.py

```diff
@@ -133,9 +133,13 @@ def check_connection():
             # Skip the sync message when unpaired because the prompt to go to
             # home.mycrof.ai will be displayed by the pairing skill
             enclosure.mouth_text(mycroft.dialog.get("message_synching.clock"))
+
         # Force a sync of the local clock with the internet
-        ws.emit(Message("system.ntp.sync"))
-        time.sleep(15)   # TODO: Generate/listen for a message response...
+        config = Configuration.get()
+        platform = config.get("enclosure", {}).get("platform", "unknown")
+        if platform in ['mark-1', 'picroft']:
```

Comment:
The official platform names are actually `mycroft_mark_1` and `picroft`. This was confusing because they are actually documented wrong in the config file (but I've created [a PR](https://github.com/MycroftAI/mycroft-core/pull/1448/files) to fix this).